### ISSUE TITLE
Implement an attractive design 

### ISSUE THREAD

**Issue by howareyouami** [itsmeAB/Todo-App#4]

**Comment by howareyouami** [comment]
How is the, please give suggestions
![Uploading Untitled-3 copy.jpg…]()

**Comment by abidhkm** [comment]
suggestion : apply full height for the vertical bars ( left of Missed tasks , Upcoming tasks etc. )

**Comment by itsmeAB** [comment]
@howareyouami Can you remove that shadow for add button ?

**Comment by howareyouami** [comment]
Yes we can remove the shadow, and apply full height for the vertical bars. Seems good.
